src/index.js

```diff
@@ -102,6 +102,11 @@ export default class Video extends WdioReporter {
     const filename = this.frameNr.toString().padStart(4, '0') + '.png';
     const filePath = path.resolve(this.recordingPath, filename);
 
+    // Create the report directory, if it does not exists
+    if (!fs.existsSync(filePath)) {
+      fs.mkdirSync(filePath);
```

Comment:
Would this also work if `filePath` has a bunch of non existing sub directories?

Reply:
My bad, instead of `filePath` we should use `this.recordingPath`.
Please refer the commit id: ea5c9e2e965d94bbf9b48f47eaf47021bb67b3aa

@christian-bromann : yes, it does create the bunch of non existing sub directories too